default/python/common/configure_logging.py

```diff
@@ -1,26 +1,124 @@
+"""
+configure_logging redirects print and the logger to use the freeorion server.
```

Comment:
log level can be set by freeorion command line option `--log-level`, by default it is DEBUG for developer version